SETUP.md

```diff
@@ -0,0 +1,57 @@
+# Setup Instructions for the following:
+
+1. Python 3.10
+2. Git
+3. Tesseract (for OCR)
+
+
+
+## Python 3.10
+
+#### Windows:
+- Download the Python 3.10 Binary File: https://www.python.org/ftp/python/3.10.0/python-3.10.0-amd64.exe
+- Follow the steps for Installation here: https://www.digitalocean.com/community/tutorials/install-python-windows-10
+
+
+#### macOS:
+- Download the Python 3.10 Package Installer: https://www.python.org/ftp/python/3.10.11/python-3.10.11-macos11.pkg
+- Follow the steps for Installation here: https://www.codingforentrepreneurs.com/guides/install-python-on-macos/
+
+
+#### Ubuntu
+- Open Terminal and type the following command:
+  - `sudo apt install python3.10-full`
+
+
+
+
+## Git
+
+#### Windows:
+- Download system appropriate binary (exe) file from here: https://git-scm.com/download/win
+- Install the setup file.
+
+
+#### macOS:
+- Follow the steps here: https://git-scm.com/download/mac
+
+
+#### Ubuntu (Linux and Unix):
+- Follow the steps here: https://git-scm.com/download/linux
+
+
+
+
+## Tesseract OCR
+
+#### Windows:
+- Download the following binary (.exe) file:
+    [Tesseract Setup](https://digi.bib.uni-mannheim.de/tesseract/tesseract-ocr-w64-setup-5.3.1.20230401.exe)
+- Run the setup file and Install the Tesseract OCR
```

Comment:
How about:

```
- Run the downloaded file to install Tesseract OCR
```

Reply:
Addressed in: [fbf264c](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/fbf264c22a4ff68818b3ef22f91049cccd96794c)